Sources/eppo/EppoClient.swift

```diff
@@ -92,6 +102,30 @@ public class EppoClient {
             }
         }
     }
+
+    public static func initialize(
+        configurationJson: String,
+        obfuscated: Bool,
+        assignmentLogger: AssignmentLogger? = nil,
+        assignmentCache: AssignmentCache? = InMemoryAssignmentCache()
+    ) throws -> EppoClient {
```

Comment:
minor: `EppoClient`'s `load()` and `loadIfNeeded()` method behave strangely/unexpectedly when client is initialized with this method.